WalletWasabi.Fluent/ViewModels/AddWallet/Create/RecoveryWordsViewModel.cs

```diff
@@ -37,6 +46,32 @@ private void OnCancel()
 		Navigate().Clear();
 	}
 
+	private async Task OnCopyToClipboardAsync()
+	{
+		if (Application.Current?.Clipboard is null)
+		{
+			return;
+		}
+
+		var words =
+			MnemonicWords.Select(x => x.Word).ToArray();
+
+		var text = string.Join(", ", words);
+
+		await Application.Current.Clipboard.SetTextAsync(text);
+
+		Observable.Timer(TimeSpan.FromSeconds(30))
```

Comment:
Let me see if I understand what goes one here :) 
The rationale here is to empty the contents of the clipboard 30 seconds after being copied, right?